gdb-xtensa-esp32-elf/build.sh

```diff
@@ -31,12 +65,26 @@ cd binutils-gdb-esp32-src
     --disable-binutils \
     --disable-ld \
     --disable-gas \
+    --without-guile \
     --disable-sim \
-    --disable-gold
+    --disable-gold \
+    --with-pkgversion="esp-gdb" \
+    --with-xtensaconfig
 make
 make install
 
+mkdir -p "$TARGET_PREFIX"/bin/
+
+# Wrapper scripts for esp32* variants
+cp "$SRC_DIR"/xtensa-esp32*-elf-gdb "$TARGET_PREFIX"/bin/
```

Comment:
Don't you need to change this to `esp` from `esp32` if you do the above change?

Reply:
these are the wrapper scripts, which are named specific to the soc variant they target; see above comment for details